packages/services/usage/src/usage.ts

```diff
@@ -268,10 +268,18 @@ export function createUsage(config: {
     status = newStatus;
   }
 
+  producer.on(producer.events.CONNECT, () => {
+    logger.info(`Kafka producer: connected`);
+  });
+
   producer.on(producer.events.REQUEST_TIMEOUT, () => {
     logger.info('Kafka producer: request timeout');
   });
 
+  producer.on(producer.events.DISCONNECT, () => {
+    logger.info(`Kafka producer: disconnected`);
```

Comment:
I want to know whenever these things happen to better understand what our kafka producer is doing when we see the spikes in memory and in general.